index.js

```diff
@@ -18,6 +18,24 @@ const manageRecipes = async () => {
     // Before adding any recipes to the database, let's remove all existing ones
     await Recipe.deleteMany();
 
+    await Recipe.create ({title: "Pasta Bolognese", cuisine: "Italian"})
+    
+    // add the whole database of recipes  (iteration 3)
+    await Recipe.insertMany(data)
+
+
+    // update recipe (iteration 4)
+
+    await Recipe.updateOne({ title: 'Rigatoni alla Genovese' }, { duration: 100 });
```

Comment:
You can use that `.updateOne()` method for updating mongoDB, however, it's better practice if you use `.findOneAndUpdate()` method. It's not wrongly used the `updateOne`, but there's small diferences between the methods and you probably are going to use more the `.findOneAndUpdate()`.